### DISCUSSION TITLE
I was asked to resubmit my onboarding, but I'm not sure which session you're referring to.

### DISCUSSION THREAD

**Discussion by Tomohiro-yasue** [microsoft/SATechnicalOnboarding#157] [Questions]
Hello Tomohiro,
 I have verified the completion of your technical onboarding and there seems to be a problem with one of the links you provided.
 The first link should lead to the online resume you created. The link should end in azurestaticapps.net/.
 Please reply here with that link so we can complete the technical onboarding process. To complete the technical onboarding, please see this resource.
 Thank you very much.
 ↑upPlease let me know from which session you have corrected or wrong

**Comment by AnthonyByansi** [comment]
Hi @Tomohiro-yasue 
It seems you hadn't deployed your site to Azure, which should eventually end with ` azurestaticapps.net/.` 

Could you please take a screenshot of the issue, perhaps I could better understand

**Reply by AnthonyByansi** [reply]
Also, please be kind to open-up an issue here: https://github.com/microsoft/SATechnicalOnboarding/issues/new/choose
